rq/dependency.py

```diff
@@ -0,0 +1,25 @@
+from .job import Job, JobStatus
+from redis.client import Pipeline
+from typing import List
+from redis.exceptions import WatchError
+
+
+class Dependency:
+    @classmethod
+    def get_ready_jobs(cls, jobs: List['Job'], pipeline: Pipeline):
```

Comment:
Can we rename `get_ready_jobs` to something more descriptive? I don't have better ideas at the moment, but `ready` is not a term that's already used in this library.

Reply:
Gave it a shot but it's very wordy, let me know if that's alright: https://github.com/rq/rq/pull/1897/commits/fd211b9ae4b06e3cb3f0b56e09f078c263a2ba3c